[BUG] Tests failing

**Describe the bug**
Tests are failing, likely due of a signature mismatch between the the tests and the code for:

* `settings.ForEach`
* `glib.IdleAdd`
* `glib.TimeoutAdd`

which do no longer take a second parameter after the function one.

This surfaced during a build of the gotk3 Debian package so I patched this. Would be happy to submit a patch upstream if the issue is confirmed. Here's the patch in Debian: https://salsa.debian.org/go-team/packages/golang-github-gotk3-gotk3/-/blob/master/debian/patches/fixtests.patch

```
...
# github.com/gotk3/gotk3/gtk [github.com/gotk3/gotk3/gtk.test]
gtk/print_test.go:51:18: too many arguments in call to settings.ForEach
	have (func(string, string), number)
	want (PrintSettingsCallback)
...
# github.com/gotk3/gotk3/glib_test [github.com/gotk3/gotk3/glib.test]
glib/glib_test.go:37:14: too many arguments in call to glib.IdleAdd
	have (func(string) bool, string)
	want (interface {})
glib/glib_test.go:51:17: too many arguments in call to glib.TimeoutAdd
	have (number, func(string) bool, string)
	want (uint, interface {})
FAIL	github.com/gotk3/gotk3/glib [build failed]
FAIL	github.com/gotk3/gotk3/gtk [build failed]
...
```

**To Reproduce**
Steps to reproduce the behavior:

```
go test -tags "glib_2_66,pango_1_42,gtk_3_20" -v ./...
```
in fact, the tags should not change anything, just needed them to make the build work on Debian buster.

**Expected behavior**
Tests should complete successfully.

**Error messages**
see above

**Environment:**
gtk3 version: '3.24.5'
go version: '1.17'
os: 'Debian buster'

--- a/glib/glib_test.go
+++ b/glib/glib_test.go
@@ -34,12 +34,11 @@ func TestConnectNotifySignal(t *testing.T) {
 		gtk.MainQuit()
 	})
 
-	glib.IdleAdd(func(s string) bool {
-		t.Log(s)
+	glib.IdleAdd(func() bool {
 		spacing++
 		box.SetSpacing(spacing)
 		return true
-	}, "IdleAdd executed")
+	})
 
 	gtk.Main()
 }
@@ -48,11 +47,10 @@ func TestConnectNotifySignal(t *testing.T) {
 func TestTimeoutAdd(t *testing.T) {
 	runtime.LockOSThread()
 
-	glib.TimeoutAdd(2500, func(s string) bool {
-		t.Log(s)
+	glib.TimeoutAdd(2500, func() bool {
 		gtk.MainQuit()
 		return false
-	}, "TimeoutAdd executed")
+	})
 
 	gtk.Main()
 }

--- a/gtk/print_test.go
+++ b/gtk/print_test.go
@@ -49,7 +49,7 @@ func TestPrintSettings(t *testing.T) {
 	settings.SetInt("Key4", 2)
 
 	settings.ForEach(func(key, value string) {
-	}, 0)
+	})
 }
 
 // TestPrintContext tests creating and manipulating PrintContext

